### ISSUE TITLE
USTVGO in The Crew has been down for 12 hours

### ISSUE THREAD

**Issue by EnderTheDestroyer** [team-crew/team-crew.github.io#82]
https://www.reddit.com/r/Addons4Kodi/comments/s2mtuv/the_crew_ustvgo_down/

**Comment by Aubreywhitney** [comment]
https://github.com/benmoose39/ustvgo_to_m3u

**Comment by anarchpenguin** [comment]
> https://github.com/benmoose39/ustvgo_to_m3u

This is great, but not helpful for the average user. They are gonna be stuck waiting for an add-on update as they do not have a clue as to how to add this to their firestick to have it all work as expected.

**Comment by Aubreywhitney** [comment]
OK.  Tell everyone to go to this website and watch all 100 channels

Https://ustvgo.tv

**Comment by Aubreywhitney** [comment]
And if you don't like the web address solution, since your already using kodi, install the PVR IPTV Simple Client that's built into kodi and go search on the web for the file ustvgo.m3u that is updated every 4 hrs and set it in simple client to repeated refresh every 120 min. You will have all channels and tv guide.

If your still crying, then provide me a free file server outside the USA and I will push/upload a current working m3u file to the server every 4 hrs 24/7 for all ustvgo channels.

If you own a NAS box, which you should have,  that has GIT and Python3 installed and you want a script file that has 4 commands to get and update m3u file every 4 hrs automatically for your personal usage I can give you that too.

Bottom line, if you want something done, don't depend on others, do it yourself. That is necessary getting something free.

**Comment by EnderTheDestroyer** [comment]
I used to think The Crew was a great addon with great support and I thought I was doing you a favor by letting someone know there was an IPTV  issue but I guess I was wrong about all of that. I'll never reach out to you again. Good day.

**Comment by fade2gray** [comment]
@EnderTheDestroyer They were most likely not a TC dev - to speak with devs you need to use the Telegram messenger for more 'informative' advice than you are like to find on a public board.

**Comment by EnderTheDestroyer** [comment]
Thanks! Not much fun having your head bitten off when you're trying to help someone.

**Comment by tonymasi** [comment]
Hey, it’s all good…Best tip ever… thanks for your work

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by EnderTheDestroyer** [comment]
Thanks Tony, much appreciated!

**Comment by anarchpenguin** [comment]
> And if you don't like the web address solution, since your already using kodi, install the PVR IPTV Simple Client that's built into kodi and go search on the web for the file ustvgo.m3u that is updated every 4 hrs and set it in simple client to repeated refresh every 120 min. You will have all channels and tv guide.
> 
> If your still crying, then provide me a free file server outside the USA and I will push/upload a current working m3u file to the server every 4 hrs 24/7 for all ustvgo channels.
> 
> If you own a NAS box, which you should have, that has GIT and Python3 installed and you want a script file that has 4 commands to get and update m3u file every 4 hrs automatically for your personal usage I can give you that too.
> 
> Bottom line, if you want something done, don't depend on others, do it yourself. That is necessary getting something free.

Bottom line is your a wanker that hasn't a clue about how the community works. Why do you not educate people instead of trying to degrade them. I for one could care less what you think, you obviously are another dumb wanker who has never worked with the open source community.

**Comment by EnderTheDestroyer** [comment]
Lol, like I care what an asshole like you thinks. Fuck off.

**Comment by fade2gray** [comment]
…and there you have it – one kicks in with the verbal abuse, then another kicks off because they think it's aimed at them when in fact it was aimed at someone else.  🤣

**Comment by EnderTheDestroyer** [comment]
I apologize to you, folks like yourself deserve better. I don't understand why people come here and attack someone who's trying to be helpful. I'm not sure why you'd think Aubreywhitney and anarchpenguin weren't aiming at me, both of their responses seemed to target me. I don't suffer bullies well, having seen my share of bullying, when they act out I usually respond in terms they'll understand and sometimes going to their level isn't very pretty. For that I apologize again.

**Comment by EnderTheDestroyer** [comment]
> > And if you don't like the web address solution, since your already using kodi, install the PVR IPTV Simple Client that's built into kodi and go search on the web for the file ustvgo.m3u that is updated every 4 hrs and set it in simple client to repeated refresh every 120 min. You will have all channels and tv guide.
> > If your still crying, then provide me a free file server outside the USA and I will push/upload a current working m3u file to the server every 4 hrs 24/7 for all ustvgo channels.
> > If you own a NAS box, which you should have, that has GIT and Python3 installed and you want a script file that has 4 commands to get and update m3u file every 4 hrs automatically for your personal usage I can give you that too.
> > Bottom line, if you want something done, don't depend on others, do it yourself. That is necessary getting something free.
> 
> Bottom line is your a wanker that hasn't a clue about how the community works. Why do you not educate people instead of trying to degrade them. I for one could care less what you think, you obviously are another dumb wanker who has never worked with the open source community.

I apologize profusely, I thought you were responding to me! I don't spend enough time on GitHub to be fluent in using it but fade2gray showed me the light. Thanks for sticking up for me, cheers!
